src/components/street-geo.js

```diff
@@ -13,16 +13,20 @@ AFRAME.registerComponent('street-geo', {
     orthometricHeight: { type: 'number', default: null },
     geoidHeight: { type: 'number', default: null },
     ellipsoidalHeight: { type: 'number', default: null },
-    maps: { type: 'array', default: [] }
+    maps: {
+      type: 'string',
+      default: 'google3d',
+      oneOf: ['google3d', 'mapbox2d']
+    }
```

Comment:
Changing the type would break existing saved json, right? Is it ok?

Reply:
@vincentfretin luckily a stringified array of 1 item appears to be identical to a string; and we didn't support choosing multiple at a time yet anyway so there shouldn't be any arrays in the db